test/web/10-CredentialSwitch.js

```diff
@@ -6,7 +6,7 @@ import {CredentialSwitch, registerComponent} from '@bedrock/vue-vc';
 import AlumniDisplay from '../components/examples/alumniDisplay.vue';
 import {config} from '@bedrock/web';
 import {createApp} from 'vue';
-import {default as iconSet} from 'quasar/icon-set/fontawesome-v5.js';
+import iconSet from 'quasar/icon-set/fontawesome-v5.mjs';
```

Comment:
weird, but ok.

Reply:
That cjs file is generated from the .mjs, so those files should be the same.  Staying with ESM seemed the way to go, and using the default export aligns with the docs.